src/js/page_sections/RelatedLinks.js

```diff
@@ -8,37 +8,55 @@ class RelatedLinks extends Component {
 
   render() {
 
-    const { relatedlinks, topicId, topicName } = this.props;
-
-    return (relatedlinks) && (
-
-      <div className="coa-section">
-        <div className="wrapper">
-          <SectionTitle title="Check out related city services" noBorder={true} />
-
-          <div className="row">
+    const { relatedlinks, sectionStyle, sectionTitle, sectionText, sectionLink } = this.props;
```

Comment:
made this page_section more generic since we have a lot of places now rendering lists of box links.